R/create.R

```diff
@@ -673,9 +672,7 @@ create_stan_delays <- function(..., ot) {
   ## map pmfs
   ret$np_pmf_groups <- array(c(0, cumsum(combined_delays$np_pmf_length)) + 1)
   ## assign prior weights
-  if (any(ret$weight == 0)) {
-    ret$weight[ret$weight == 0] <- ot
-  }
+  ret$weight <- array(rep(weight, ret$n_p))
```

Comment:
@seabbs on your question of variable name change: in stan it's still a per-delay weight (called `weight`) which is set here. It's only that we don't pass `ot` to this but rather an explicit weight (as it's only used for that) which is effectively set to `ot` in the `estimate_` functions.

Reply:
Got it thanks